managed/src/main/resources/v1.routes

```diff
@@ -148,7 +148,7 @@ POST    /customers/:cUUID/universes/:uniUUID/upgrade                           c
 POST    /customers/:cUUID/universes/:uniUUID/disk_update                       com.yugabyte.yw.controllers.UniverseController.updateDiskSize(cUUID: java.util.UUID, uniUUID: java.util.UUID)
 GET     /customers/:cUUID/universes/:uniUUID/status                            com.yugabyte.yw.controllers.UniverseController.status(cUUID: java.util.UUID, uniUUID: java.util.UUID)
 GET     /customers/:cUUID/universes/:uniUUID/health_check                      com.yugabyte.yw.controllers.UniverseController.healthCheck(cUUID: java.util.UUID, uniUUID: java.util.UUID)
-GET     /customers/:cUUID/universes/find/:universeName                         com.yugabyte.yw.controllers.UniverseController.findByName(cUUID: java.util.UUID, universeName: String)
+GET     /customers/:cUUID/universes/find/:universeName                         com.yugabyte.yw.controllers.UniverseController.findByName_Corrected(cUUID: java.util.UUID, universeName: String)
```

Comment:
is this change related to the AZ rendering issue?

Reply:
No, This is an additional request to change the API of the universe name, for now, it is checking the name and if it got a response other than 200 then it says that Universe already exists.

Reply:
Actually leave out this change. If UI is not going to use this then i can do with other related changes.

Reply:
Leave this as is. Instead rename findByName_Corrected as suggested above.